docs/source/index.rst

```diff
@@ -0,0 +1,11 @@
+Obal: Packaging Wrapper Using Ansible
+=====================================
+
+Obal is named after the Czech word for wrapper or packaging. It is a command line tool aimed to provide a unified packaging workflow powered by Python and Ansible.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   playbook
```

Comment:
`playbook.rst` is about writing actions, so I think it should be a subchapter of `development.rst`.